### ISSUE TITLE
WNLI Preprocess and Finetune Code

### ISSUE THREAD

**Issue by tobydu** [pytorch/fairseq#2006]
Could you please provide or point me to the code for WNLI preprocessing and finetuning? I am trying to reproduce the result in the paper but couldn't seem to get the accuracy because `WNLI` is not part of `TASKS="QQP MNLI QNLI MRPC RTE STS-B SST-2 CoLA"`.

**Comment by myleott** [comment]
For WNLI we used the original "WSC" data in SuperGLUE. Instructions to reproduce our results can be found here: https://github.com/pytorch/fairseq/tree/master/examples/roberta/wsc
